*** usr/bin/pg_buildext	Mon Dec 16 11:43:17 2024
--- usr/bin/pg_buildext	Mon Dec 16 11:48:46 2024
***************
*** 107,117 ****
  install() {
      prepare_env $1
      package=`echo $opt | sed -e "s:%v:$1:g"`
  
      mkdir -p $vtarget
      # if a Makefile was created by configure, use it, else the top level Makefile
      [ -f $vtarget/Makefile ] || makefile="-f $srcdir/Makefile"
!     make -C $vtarget ${makefile:-} install DESTDIR="$PWD/debian/$package" PG_CONFIG="$pgc" VPATH="$srcdir" USE_PGXS=1 ${MAKEVARS:-} || return $?
      substvars "$1" "$package"
  }
  
--- 107,119 ----
  install() {
      prepare_env $1
      package=`echo $opt | sed -e "s:%v:$1:g"`
+     procs="$(nproc)"
+     mjobs="$(expr $procs + 1)"
  
      mkdir -p $vtarget
      # if a Makefile was created by configure, use it, else the top level Makefile
      [ -f $vtarget/Makefile ] || makefile="-f $srcdir/Makefile"
!     make -j${mjobs} -C $vtarget ${makefile:-} install DESTDIR="$PWD/debian/$package" PG_CONFIG="$pgc" VPATH="$srcdir" USE_PGXS=1 ${MAKEVARS:-} || return $?
      substvars "$1" "$package"
  }
  
